### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by ercling**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+app52:
```
Comment (andreinwald) [inline docker-compose.yml]: Docker config no need in repo

File: mailfire/Mailfire.php [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
 
-namespace Mailfire;
-
 class Mailfire
 {
     public $clientId;
     public $clientKey;
 
-    public function __construct($clientId, $clientKey)
+    public function __construct($clientId, $clientKey, $throwExceptions = false)
```
Comment (andreinwald) [inline mailfire/Mailfire.php]: $throwExceptions need as variable in class. Not param

File: docker-compose.yml (hunk 1 above)
Comment (ercling) [inline docker-compose.yml]: Removed

File: mailfire/Mailfire.php (hunk 2 above)
Comment (ercling) [inline mailfire/Mailfire.php]: Changed

**Pull request merged**